CHANGELOG.md

```diff
@@ -7,7 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.37.0](https://github.com/Substra/substra/releases/tag/0.37.0) - 2022-09-19
+
 ### Changed
+
 - BREAKING CHANGE: `schemas.ComputePlanSpec` `clean_models` property is now removed, the transient property on tasks outputs should be used instead.
 
```

Comment:
we said that we would add all changes to the changelog 

[tests: fix algo inputs and task code consistency](https://github.com/Substra/substra/pull/278)